ksql-parser/src/test/java/io/confluent/ksql/util/QueryMaskTest.java

```diff
@@ -250,4 +250,38 @@ public void shouldMaskMixedValidInvalidStatements() {
 
     assertThat(maskedQuery, is(expected));
   }
+
+  @Test
+  public void shouldMaskInsertStatement() {
+    // Given
+    final String query = "--this is a comment. \n"
+        + "INSERT INTO foo (KEY_COL, COL_A) VALUES (\"key\", \"A\");";
+
+    // When
+    final String maskedQuery = QueryMask.getMaskedStatement(query);
+
+    // Then
+    final String expected = "INSERT INTO `FOO` (`KEY_COL`, `COL_A`) VALUES ('[value]', '[value]');";
+
+    assertThat(maskedQuery, is(expected));
+  }
+
+  @Test
+  public void shouldMaskFallbackInsertStatement() {
```

Comment:
Is this test still exercising the fallback masking logic? I was expecting we'd need to change the query string in order to have it exercise the fallback logic through `QueryMask.getMaskedStatement()`.

Reply:
Hey Victoria, yes I made sure the test below is throwing an exception (2 columns defined but 6 values inserted) thus testing the fallback logic here

Reply:
Ooh interesting. I'm surprised that that causes the statement to not parse successfully -- I would've expected it to cause the statement to parse successfully but be rejected by the executor later on, and therefore still use the parse-tree-based masking logic rather than the fallback logic. If you've tested that it's using the fallback logic, then that sounds good to me.